### DISCUSSION TITLE
Cost control

### DISCUSSION THREAD

**Discussion by danilocasabona** [aws-solutions/workload-discovery-on-aws#560] [General]
Hey folks,

I am wondering if it is possible to control cost with this amazing tool. My ideia is to have it ready to go and pay only when we run a new assessment or during pre configured periods. Make sense? Does someone know if it possible?

**Comment by aws-ymson** [comment]
Closing this out: Workload Discovery on AWS has been deprecated as of 08/14/2026 and is no longer being maintained.
